src/libs/ReportUtils.js

```diff
@@ -1503,12 +1507,15 @@ function buildOptimisticExpenseReport(chatReportID, policyID, payeeAccountID, to
  */
 function getIOUReportActionMessage(type, total, comment, currency, paymentType = '', isSettlingUp = false) {
     const currencyUnit = CurrencyUtils.getCurrencyUnit(currency);
-    const amount = NumberFormatUtils.format(preferredLocale, total / currencyUnit, {style: 'currency', currency});
+    const amount = NumberFormatUtils.format(preferredLocale, Math.abs(total) / currencyUnit, {style: 'currency', currency});
     let paymentMethodMessage;
     switch (paymentType) {
         case CONST.IOU.PAYMENT_TYPE.ELSEWHERE:
             paymentMethodMessage = ' elsewhere';
             break;
+        case CONST.IOU.PAYMENT_TYPE.VBBA:
```

Comment:
What about `PAYMENT_TYPE.EXPENSIFY`?

Reply:
The payment type is `Expensify` so this will Just Work 

`using ${paymentType}`;